ws-nextjs-app/cypress/e2e/liveTvPage/index.cy.ts

```diff
@@ -0,0 +1,23 @@
+import { LIVE_TV_PAGE } from '#app/routes/utils/pageTypes';
+import { assertLiveTvPageTest, assertLiveTvPageLocal } from './assertions';
+import runTestsForPage from '../../support/helpers/runTestsForPage';
+
+const testSuites = [
+  {
+    path: 'dari/watch/bbc_afghan_tv/live',
+    service: 'ws',
+    runforEnv: ['local'],
+    tests: [assertLiveTvPageLocal],
+  },
+  {
+    path: 'dari/watch/bbc_afghan_tv/live',
+    service: 'ws',
+    runforEnv: ['test'],
+    tests: [assertLiveTvPageTest],
+  },
```

Comment:
These can also be combined
```suggestion
  {
    path: '/dari/watch/bbc_afghan_tv/live',
    service: 'dari',
    runforEnv: ['local', 'test'],
    tests: [assertLiveTvPageTitle],
  }
```